FirebaseFunctions/Tests/SwiftIntegration/IntegrationTests.swift

```diff
@@ -0,0 +1,258 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+import FirebaseFunctions
+import FirebaseFunctionsTestingSupport
+import XCTest
+
+class IntegrationTests: XCTestCase {
+  let functions = FunctionsFake(
+    projectID: "functions-integration-test",
+    region: "us-central1",
+    customDomain: nil,
+    withToken: nil
+  )
+  let projectID = "functions-swift-integration-test"
+
+  override func setUp() {
+    super.setUp()
+    functions.useLocalhost()
+  }
+
+  func testData() {
+    let expectation = self.expectation(description: #function)
```

Comment:
nit: `self` is not required here, can be just `expectation(description: #function)`.